backend/app/entity/toggle.go

```diff
@@ -1,7 +1,14 @@
 package entity
 
+import "database/sql"
```

Comment:
No SQL in either entity or use case. They are core business logics. Should not depends on specific technology. Let's use enum instead: https://blog.learngoprogramming.com/golang-const-type-enums-iota-bc4befd096d3